src/NServiceBus.Persistence.DynamoDB/Utility/NonNativePubSubCheck.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace NServiceBus.Persistence.DynamoDB
+{
+    using System;
+    using NServiceBus.Features;
+
+    class NonNativePubSubCheck
+    {
+        public static void ThrowIfMessageDrivenPubSubInUse(FeatureConfigurationContext context)
+        {
+#pragma warning disable CS0618 // Type or member is obsolete - Remove this method along with MessageDrivenSubscriptions
+            if (context.Settings.IsFeatureEnabled(typeof(MessageDrivenSubscriptions)) || context.Settings.IsFeatureActive(typeof(MessageDrivenSubscriptions)))
```

Comment:
Then this would throw. I was assuming because we target v8 that would be fine but maybe we should remove this check because we still have the compat mode for SQS in v8 right?

Reply:
> we still have the compat mode for SQS in v8 right?

Correct

Reply:
Fixed